presto-tests/src/test/java/com/facebook/presto/tests/TestTpchLocalStats.java

```diff
@@ -120,4 +122,125 @@ public void testVarcharComparisons()
         statisticsAssertion.check("SELECT * FROM orders WHERE 'this is always ...' = '... false'",
                 checks -> checks.estimate(OUTPUT_ROW_COUNT, noError()));
     }
+
+    @Test
+    public void testInnerJoinStats()
```

Comment:
can you add tests for inequality joins and cross joins for all of these.  And also for multiple join keys for the outer joins.

Reply:
@sopel39 ?

Reply:
> can you add tests for inequality joins and cross joins for all of these

Added tests with `true/false` predicate. Inequality joins testing here would rather introduce some noise because filter computations are more complex.